tethysext/atcore/templates/atcore/app_users/modify_resource.html

```diff
@@ -103,6 +103,7 @@ <h1>{% if editing %}Edit{% else %}New{% endif %} {{ type_singular }}</h1>
   <link href="{% static 'atcore/file_input/css/file_input.css' %}" rel="stylesheet"/>
   {% endif %}
   <link href="{% static 'atcore/css/form_processing.css' %}" rel="stylesheet"/>
+  <link href="{% static 'atcore/css/center.css' %}" rel="stylesheet"/>
```

Comment:
Is center.css needed here? I don't see any of the classes that are used to center things being used in the changes.

Reply:
You are right. that was not being used so I removed it. I also found a resize bug in the spatial reference droupdown of this template and took care of it.

Reply:
Actually, center.css is used to display the loading bar so I've put it back. See comment below for more info.